src/types.ts

```diff
@@ -12,18 +11,13 @@ export interface Question {
   max_char_limit: number | null;
```

Comment:
`solution` and all not needed here?

Reply:
will update it once we connect it to the actual backend, etc.